### PR TITLE
update data augmentation result

### PR DESCRIPTION
**Pull request by huqiwen0313**
add data augmentation results to czi-report

### CODE REVIEW

**Comment by cgreene** [comment]
@huqiwen0313 : I think your resolution of the merge conflict resulted in removing the section you just added. I can't add messages since it no longer shows up as a changed file. Can you try to put those changes back in?

**Comment by huqiwen0313** [comment]
I added the results back.  Let me know if there is still an issue. Thanks !

**Comment by cgreene** [comment]
I'm also not sure the figures made it back in with the commit. May want to check.

**Comment by huqiwen0313** [comment]
@cgreene, I modified the text, can you take a look at it ? I think the figures should be fine, I added them into the image folder.

**Comment by cgreene** [comment]
Ahh - the figs don't appear to be part of this PR so I was confused. This has a typo for NMI on the axis: https://github.com/greenelab/czi-hca-report/blob/master/content/images/kmeans.data.augmentation.png

**Comment by huqiwen0313** [comment]
@cgreene, can you please take a look at if the text is good ? If so, I'll merge this pull request, thanks !

File: content/03.aim1.md [hunk 1]
```diff
@@ -124,3 +124,29 @@ The performance of VAE is generally better than the other approaches for majorit
...
+We hypothesized that genomic data augmentation will improve latent feature generalization by separating biological from technical features and increasing the effective sample size during training.
+
+We performed data augmentation based on HCA melanoma dataset. 
+We down sampled the melanoma dataset to test the method stability.
+The dataset was bootstrapped with 100, 500, 1000 and 2000 cells. 
+We expected data augmentation will contribute more for small number of cells rather than large number of cells.
+Salmon (version 0.8.1) [@doi:10.1038/nmeth.4197] was used to do the transcript abundance quantification and we resampled the reads by 2x, 5x and 10x times to capture the uncertainty in the data. 
+
+Figure 7 and 8 show the performance of data augmentation under different conditions. 
+K-means and Knn based performance both indicate data augmentation works well in small number of samples (100 and 500 cells), but does not contribute too much in a large sample size (2000).
```
Comment (cgreene) [inline content/03.aim1.md]: Maybe: "... works well for very small numbers of cells (100), but does not appear to help when the number of cells is modest (2000).

**Comment by cgreene** [comment]
I have one requested change - after making that change, feel free to merge!

**Pull request merged**